Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,218 @@
   - SurveillanceWirelessCameraMonitorCircuitboard
   - ComputerTelevisionCircuitboard
 
-#- type: technology
-#   name: "ballistic technology"
-#   id: BallisticTechnology
-#   description: Just a fancy term for guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Pistols/mk58.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
-#
-# - type: technology
-#   name: "direct energy technology"
-#   id: DirectEnergyTechnology
-#   description: Basically laser guns.
-#   icon:
-#     sprite: Objects/Weapons/Guns/Battery/taser.rsi
-#     state: icon
-#   requiredPoints: 15000
-#   requiredTechnologies:
-#     - SecurityTechnology
+- type: technology
+  name: technologies-security-basic
+  id: BasicSecurityTechnology
+  description: technologies-security-basic-description
+  icon:
+    sprite: Objects/Misc/handcuffs.rsi
+    state: handcuffs
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicResearch
+  unlockedRecipes:
+    - Handcuffs
+    - Zipties
+    - StunBaton
+    - RiotShield
+    - RiotLaserShield
+    - RiotBulletShield
+    - Flash
+    - FlashPayload
+    - ForensicPad
+    - ForensicScanner
+    - ClothingGlassesEyesSecurity
+    - FlashlightSeclite
+
+- type: technology
+  name: technologies-ballistic-ammunition
+  id: BallisticAmmunitionTechnology
+  description: technologies-ballistic-ammunition-description
+  icon:
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    state: base-6
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - MagazineBoxPistol
+    - MagazineBoxPistolRubber
+    - MagazineBoxMagnum
+    - MagazineBoxMagnumRubber
+    - MagazineBoxRifle
+    - MagazineBoxRifleRubber
+    - MagazineBoxLightRifle
+    - MagazineBoxLightRifleRubber
```

Comment:
printing entire boxes of ammunition seems pointless when you have magazines. Also, 5000 is too cheap.